sys-fs/udiskie/udiskie-1.7.5.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_4 python3_5 python3_6 )
+inherit distutils-r1 gnome2-utils
+
+DESCRIPTION="An automatic disk mounting service using udisks"
+HOMEPAGE="https://pypi.org/project/udiskie/ https://github.com/coldfix/udiskie"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE=""
+
+RDEPEND="dev-python/docopt[${PYTHON_USEDEP}]
```

Comment:
`setup.py` also lists `gbulb` as a requirement. Are you sure it works without it?

Reply:
Actually it does for me, but I wonder why - obviously I don't get in contact with that code with my use of udiskie. I see gbulb requires python 3.5+, so 3.4 needs to go. I guess I'll have to package gbulb as well then :)

Reply:
FWICS it's used in `cli.py`.